lib/android/audio.nit

```diff
@@ -14,11 +14,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Android audio services
+# Android audio services, wrapps a part of
+# http://developer.android.com/reference/android/media/package-summary.html
```

Comment:
*wraps

Do not use an URL in the synopsis. You can put it at the end of the doc, this information is not so important.